trm/setting_wcp_arg.tex

```diff
@@ -0,0 +1,79 @@
+\[
+    \boxed{\text{All constraints in this subsection assume } \first_{i} = 1 }
+\]
+To this end We impose the following constraints
+\begin{description}
+	\def\nRows{\rowOffsetAddress}\item[\underline{Processing row $n^\circ(\nRows)$:}] 
+		we impose that
+		\[
+			\wcpCallToLt {
+				anchorRow = i                      ,
+				relOffset = \nRows                 ,
+				argOneHi  = \trmAddrHi             ,
+				argOneLo  = \rawAddrLo             ,
+				argTwoHi  = (256 << 5)             ,
```

Comment:
Why 5 ? Also `bla << 1` typically means `2 * bla` i.e. it's a single left bit shift. Regardless of what `bla` is. So we must either write `(1 << 32)` or `256 ^ 4`.